openedx/core/lib/token_utils.py

```diff
@@ -104,6 +107,7 @@ def attach_profile_claim(self, payload):
 
     def encode(self, payload):
         """Encode the provided payload."""
+        set_custom_metric('jwt_asymmetric', self.asymmetric)
```

Comment:
Reminder to update this page with any new metrics: https://openedx.atlassian.net/wiki/spaces/PERF/pages/54362736/Custom+Metrics+in+New+Relic

Reply:
updated.